packages/openapi-v2/test/unit/controller-spec/controller-decorators.test.ts

```diff
@@ -371,4 +358,10 @@ describe('Routing metadata', () => {
       in: 'query',
     });
   });
+
+  function givenAnOperationSpec() {
+    return anOperationSpec()
+      .withStringResponse()
```

Comment:
hmm...I think if this helper function calls `withStringResponse` then we probably could not omit the `return` in those controller functions.

Reply:
Discussed this with Janny, but for posterity, we do not have any assertions in the tests for the response object or the actual response, and thus having a return statement with a string for the controller methods doesn't have any effect.